content/applications/inventory_and_mrp/manufacturing/advanced_configuration/sub_assemblies.rst

```diff
@@ -65,12 +90,123 @@ order for the main product, there are two options:
   under the :guilabel:`Inventory` tab of the sublevel product's product form.
 
 .. seealso::
-   :doc:`../../purchase/products/reordering`
-   :doc:`../../inventory/warehouses_storage/replenishment/mto`
+   `<../../inventory/warehouses_storage/replenishment/reordering_rules>`
+   `<../../inventory/warehouses_storage/replenishment/mto>`
```

Comment:
Missing the role tag, and no need to include the doc path in `<...>` when not including link text, as the doc reference will use the referenced doc's H1 by default

<img width="677" alt="Screenshot 2025-03-14 at 11 34 28 AM" src="https://github.com/user-attachments/assets/d437269a-21e5-486d-82d2-15370b2de15f" />

```suggestion
   - :doc:`../../inventory/warehouses_storage/replenishment/reordering_rules`
   - :doc:`../../inventory/warehouses_storage/replenishment/mto`
```